webcompat/static/css/development/components/comment.css

```diff
@@ -161,7 +166,6 @@
 
   /* QR */
   .wc-Comment-qr {
-    position:absolute;
     z-index: 1;
-    right: 0;
+    margin: -6px 16px 0 auto;
```

Comment:
hey, magic number here ?

You can remove those two line,  Because we can add a padding directly `wc-Comment-header`